taskcluster/scripts/toolchain/android-gradle-dependencies/after.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash -vex
```

Comment:
This is copy of https://searchfox.org/mozilla-central/rev/2cd2d511c0d94a34fb7fa3b746f54170ee759e35/taskcluster/scripts/misc/android-gradle-dependencies/after.sh. I removed `gradle-plugins` which doesn't seem to be used in this project.

Reply:
Okay. Done!